pkg/ccl/cliccl/load.go

```diff
@@ -122,6 +132,35 @@ func runLoadShowSummary(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func runLoadShowBackups(cmd *cobra.Command, args []string) error {
+
+	path := args[0]
+	if !strings.Contains(path, "://") {
+		path = cloudimpl.MakeLocalStorageURI(path)
+	}
+	ctx := context.Background()
+	store, err := cloudimpl.ExternalStorageFromURI(ctx, path, base.ExternalIODirConfig{},
+		cluster.NoSettings, newBlobFactory, security.RootUserName(), nil /*Internal Executor*/, nil /*kvDB*/)
+	if err != nil {
+		return err
+	}
+
+	backupPaths, err := store.ListFiles(ctx, "/*/*/*/"+backupccl.BackupManifestName)
```

Comment:
nit: Could we move this into a very small method inside `backupccl` (maybe in `backup_destinations`?) that keeps the internal format of the backup inside `backupccl`. I'm thinking something like `backupccl.ListFullBackupsInCollection(ctx, store)`. Although it's a small wrapper, I would like to keep as much of the specifics of how backups are stored inside `backupccl` and limit visibility into the backup format with a more structured API.